### PR TITLE
Review/editing pass on the pt-br text of the getting-started section

### PR DESCRIPTION
**Pull request by vkorbes**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [x] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

#### Description
This is a review/editing pass on the pt-br text of the getting-started section. I focused on making the text sound more natural, how Portuguese actually does, instead of making sure every word was perfectly translated from the original.

I'm getting a mountain of errors when trying to run the tests. Since I didn't change the structure of the file in any way though, only words and sentences, I don't think the errors have anything to do with my changes. Lemme know if I'm wrong.

### CODE REVIEW

**Review by frontendwizard — APPROVED** [review]
LGTM. Definitely improves the translation. Good work @ellenkorbes

File: seed/challenges/00-getting-started/getting-started.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
         ],
         [
           "//i.imgur.com/vLNso6h.jpg",
-          "An illustration showing that you will learn HTML5, CSS3, JavaScript, Databases, Git, Node.js, React and D3.",
+          "An illustration showing that you will learn HTML5, CSS3, JavaScript, Databases, Git, Node.js, React, and D3.",
```
Comment (systimotic) [inline seed/challenges/00-getting-started/getting-started.json]: The next sentence does not have an Oxford comma either. Can you remove the added Oxford comma here for consistency?

**Review by systimotic — CHANGES_REQUESTED** [review]
Thanks a lot for these improvements @ellenkorbes! Big thanks to @julianorafael as well for QA'ing!
We try to get PR's reviewed and merged as soon as we can, but we've been [really busy](https://github.com/freeCodeCamp/freeCodeCamp/pulse) since the beta release. Things can take a little longer sometimes 😅

**Review by systimotic — APPROVED** [review]
# LGTM 🎉 
Thanks for the improvements @ellenkorbes!
Again, a big thanks to @julianorafael as well for QA'ing, having another speaker QA is crucial to great translations.

**Pull request merged**